docs/introduction/basics-about-model-architecture.md

```diff
@@ -67,15 +67,15 @@ class Product
 }
 ```
 
-### Factory
+## Factory
```

Comment:
It was indented on purpose - it's a factory for entities, so it should be a child of the *Entity* heading: 
- Basics about the model
- Entity
  - Example
  - Factory
    - Example
- Repository
  - Example
- Facade
  - Example
- Service
  - Example
- Cooperation of layers

It would make sense to either leave it indented, or at least renaming it to *Entity Factory*.